### PR TITLE
Add configurable reuseIdentifier

### PR DESCRIPTION
**Pull request by davideme**

### CODE REVIEW

File: Example/Example/ComicsViewController.swift [hunk 1]
```diff
@@ -19,6 +19,12 @@ class ComicsViewController: ExampleViewController, BothamCollectionViewControlle
     override func viewDidLoad() {
         collectionView.accessibilityLabel = "Comics CollectionView"
         collectionView.delegate = self
+        dataSource.reuseIdentifierForItem = { item in
+            if case .Featured = item {
+                return "FeaturedComicCollectionViewCellReuseIdentifier"
+            }
+            return "ComicCollectionViewCellReuseIdentifier"
+        }
```
Comment (pedrovgs) [inline Example/Example/ComicsViewController.swift:27]: Is this why we want to implement something like: https://github.com/pedrovgs/Renderers? Or is this approach enough?

Comment (davideme) [inline Example/Example/ComicsViewController.swift:27]: It's a start to a generic solution, to give functionality similar to
renderers.

Comment (pedrovgs) [inline Example/Example/ComicsViewController.swift:27]: Then we should extract this binding to another class. If we want to reuse this configuration in another view controller. For this case is not needed, but in the future we will want to reuse this configuration in different table views in the same app. If you want, we can merge this branch and talk about hot to improve this part in the future.

**Comment by pedrovgs** [comment]
Should we add some tests to the project to check if this feature is working as expected?

**Comment by davideme** [comment]
Yeah, of course we should have test, but I think I will try to implement a more complex example first, it's seems to simple to really be useful. It's more something that fit that my implementation that somenthing than a real example that is abstract into the library.

**Comment by pedrovgs** [comment]
Ok, waiting for it :) Thanks!

**Comment by davideme** [comment]
We will close this proposal until future real case need.

**Pull request closed without merging**